new CI failure on main: test_make_forward_dipole

```
 ___________________________ test_make_forward_dipole ___________________________
mne/forward/tests/test_make_forward.py:422: in test_make_forward_dipole
    assert_allclose(dip_fit.pos, dip_test.pos, rtol=0, atol=1e-2,
E   AssertionError: 
E   Not equal to tolerance rtol=0, atol=0.01
E   position mismatch
E   Mismatched elements: 1 / 9 (11.1%)
E   Max absolute difference: 0.01529636
E   Max relative difference: 0.59013717
E    x: array([[ 0.041216,  0.018087,  0.09476 ],
E          [ 0.044785,  0.021239,  0.050516],
E          [ 0.037098, -0.004066,  0.060205]])
E    y: array([[ 0.02592,  0.01772,  0.09169],
E          [ 0.04728,  0.02485,  0.05055],
E          [ 0.03734, -0.00548,  0.05957]])
```

Has shown up in 2 unrelated PRs today.

--- a/mne/forward/tests/test_make_forward.py
+++ b/mne/forward/tests/test_make_forward.py
@@ -393,7 +393,7 @@ def test_make_forward_dipole(tmp_path):
     # Now simulate evoked responses for each of the test dipoles,
     # and fit dipoles to them (sphere model, MEG and EEG)
     times, pos, amplitude, ori, gof = [], [], [], [], []
-    nave = 200  # add a tiny amount of noise to the simulated evokeds
+    nave = 400  # add a tiny amount of noise to the simulated evokeds
     for s in stc:
         evo_test = simulate_evoked(fwd, s, info, cov,
                                    nave=nave, random_state=rng)

